Dockerfile.gcc

```diff
@@ -135,9 +135,16 @@ RUN make O=/kernel-out LOCALVERSION="-${LOCALVERSION}" M=/tmp/hailo/linux/pcie -
     make O=/kernel-out LOCALVERSION="-${LOCALVERSION}" M=/tmp/hailo/linux/pcie -C /kernel-src INSTALL_MOD_STRIP=1 \
     INSTALL_MOD_PATH=/tmp/kernel-modules modules_install
 
+# we do not use Makefile and directly build out-of-tree module
+# PR https://github.com/enlyze/xr-usb-serial/pull/5 to make driver compatible
+# with kernel < 6.6 was merged
+ADD --keep-git-dir=true https://github.com/enlyze/xr-usb-serial.git /tmp/uart
```

Comment:
@rucoder, could you please ask the maintainer to create a TAG for this version so it can be pointed here.... we had issues before using master branch for the Realtek WiFi driver for Jetson devkit board, eventually a commit was introduced to master and broke our build.... so let's fix the revision to a known working version....

Reply:
@rene I will specify a commit id when the work is completed.